recipes/fuse-overlayfs/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set version = "1.2.0" %}
+
+package:
+  name: fuse-overlayfs
+  version: {{ version }}
+
+source:
+  - url: https://github.com/containers/fuse-overlayfs/archive/v{{ version }}.tar.gz
+    sha256: 5df218732244059057686194b0e1fef66fb822d4087db48af88e1bc29bb1afde
+    patches:
+      - 0001-Add-missing-defines-for-CentOS-6.patch
+      - 0002-Fix-version-tag.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - autoconf >=2.69
+    - automake >=1.9
+    - make
+    - pkg-config >=0.9.0
+  host:
+    - libfuse >=3.2.1
+  run:
+    - libfuse >=3.2.1
```

The pin here will cause potentially cause ABI incompatible versions to be installed. This can happen as `libfuse 3.10.0` can be used for the build while still letting a older version (e.g. `libfuse 3.2.1`) be used at install time. As the ABI looks very stable on ABI tracker:

```suggestion
    - {{ pin_compatible('libfuse') }}
```

I was about to ask for run_exports to be added to the feedstock but it seems you already have (https://github.com/conda-forge/libfuse-feedstock/pull/5)!